cmd/downloader/bucket/bucket.go

```diff
@@ -40,27 +45,36 @@ func GenerateArtifactURL(project, version, fileName string) string {
 
 // GetArtifactInfo generates a structure of all necessary information
 // from the Google Cloud Storage bucket
-func GetArtifactInfo(platform, architecture, release string, service types.Service) *types.ArtifactInfo {
+func GetArtifactInfo(platform, architecture, release string, service *types.Service) *types.ArtifactInfo {
 	if len(service.Release) == 0 {
 		glog.Fatalf("Bucket type strategy requires a branch name as `release` value. Found %s at root", release)
 		panic("")
 	}
 
 	project := service.Strategy.Project
 	release = utils.CleanBranchName(service.Release)
+	buildInfo, err := GetBuildInformation(release, project)
+	if err != nil {
+		glog.Fatal(err)
+	}
 
 	var info = &types.ArtifactInfo{
 		Name:         service.Name,
 		Platform:     platform,
 		Architecture: architecture,
-		Version:      GetArtifactVersion(release, project),
+		Version:      GetArtifactVersion(*buildInfo),
 	}
+
 	extension := utils.PlatformExt(platform)
 	packageName := fmt.Sprintf("livepeer-%s", service.Name)
 	if len(service.Binary) > 0 {
 		packageName = service.Binary
 	}
 	info.ArchiveFileName = fmt.Sprintf("%s-%s-%s.%s", packageName, info.Platform, info.Architecture, extension)
+	// if buildInfo.Builds != nil && service.SrcFilenames == nil {
+	// 	service.SrcFilenames = buildInfo.Builds
+	// }
+
```

Comment:
Should be removed?

Reply:
I was thinking of modifying the `Builds` attribute, but adding a new attribute seems easier. Fixed in next commit :)